services/eco2.md

```diff
@@ -0,0 +1,25 @@
+# Equivalent CO²
+
+Measures equivalent CO² levels.
+
+    identifier: 0x169c9dc6
+    extends: _sensor
+    camel: eCO2
+
+## Registers
+
+     ro e_CO2: u16 ppm { typical_min=400, typical_max=8192 }  @ reading
```

Comment:
should we do u32? or maybe u22.10? in general, there isn't much advantage using u16/u8 readings for sensors given all the other overheads in the system

Reply:
use u32

Reply:
use u22.10